docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -61,7 +61,7 @@ To add a helpful review, please:
 Opening a new issue and/or PR
 =============================
 
-If the problem you have noticed is too complex to fix with the ``Edit on GitHub`` option, and no open issue or PR already documents the problem, please open an issue and/or a PR on the correct underlying repo - ``ansible/ansible`` for most pages that are not plugin or module documentation. If the documentation page has no ``Edit on GitHub`` option, check if the page is for a module within a collection. If so, follow the link to the collection on Galaxy and select the ``repo`` button in the upper right corner to find the source repository for that collection and module. The Collection README file should contain information on how to contribute to that collection, or report issues.
+If the problem you have noticed is too complex to fix with the ``Edit on GitHub`` option, and no open issue or PR already documents the problem, please open an issue and/or a PR on the correct underlying repo - ``ansible/ansible-documentation`` for most pages that are not plugin or module documentation. If the documentation page has no ``Edit on GitHub`` option, check if the page is for a module within a collection. If so, follow the link to the collection on Galaxy and select the ``repo`` button in the upper right corner to find the source repository for that collection and module. The Collection README file should contain information on how to contribute to that collection, or report issues.
```

Comment:
Coming in late here but we should update this to suggest they navigate up to the collection page on docs.ansible.com. All of the collections have links to the issue tracker:
![image](https://github.com/ansible/ansible-documentation/assets/30267855/7608ff03-731e-4b73-9b3f-4155dff90913)